docs/system_overview/configuration.md

```diff
@@ -14,65 +14,305 @@ This scrambled password file is saved after an `iinit` is run. If this file does
 
 This file defines the behavior of the server Agent that answers individual requests coming into iRODS. It is created and populated by the installer package.
 
-This file contains the following top level entries:
-
-  - `advanced_settings` (required) - Contains subtle network and password related variables.  These values should be changed only in concert with all connecting clients and other servers in the Zone.
-
-    - `default_log_rotation_in_days` (default 5) - The number of days the log is written to before being rotated out.
+!!! IMPORTANT
+    All configuration options are required unless marked as optional.
 
-    - `default_number_of_transfer_threads` (required) (default 4) - The number of threads enabled when parallel transfer is invoked.
+```json
+{
+    // Defines the filename of JSON schema used for validation when not specified explicitly.
+    "schema_name": "server_config",
 
-    - `default_temporary_password_lifetime_in_seconds` (required) (default 120) - The number of seconds a server-side temporary password is good.
+    // Defines the version of the schema used for validation.
+    // This normally maps to a directory under /var/lib/irods/configuration_schemas.
+    "schema_version": "v4",
+    
+    // Contains subtle network and password related variables.
+    // These values should be changed only in concert with all connecting clients and other servers in the Zone.
```

Comment:
password?

some of these settings are definitely this-server-only, so don't require 'in concert' edits elsewhere.

perhaps this note is mentioned on each setting where this is true.